dashboard/sample_metadata_classifier.py

```diff
@@ -301,5 +301,16 @@ def interpret(project, papers, bits):
                 location="Copenhagen",
                 date="2015-04",
                 collection="saliva")
+    elif project in papers["Prussin 2019"]["projects"]:
+        sample, na_type, date, description, season = bits
```

Comment:
If you're calling this sampling_range in the final dict, call it sampling_range all the way through

Reply:
done